webrtc.html

```diff
@@ -180,7 +180,7 @@ <h2>
         The term <dfn>media transport</dfn> is defined in [[!RFC7656]].
       </p>
       <p>
-        The term <dfn>generation</dfn> is defined in [[!TRICKLE-ICE]] Section
+        The term <dfn>generation</dfn> is defined in [[RFC8838]] Section
```

Comment:
Should this be normative?

Reply:
by default, references from normative sections are normative - they no longer need to be annotated with a `!`